packages/flutter_test/lib/src/binding.dart

```diff
@@ -1179,8 +1179,8 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   }
 }
 
-/// A variant of [TestWidgetsFlutterBinding] for executing tests in
-/// the `flutter test` environment.
+/// A variant of [TestWidgetsFlutterBinding] for executing tests typically
+/// the `flutter test` environment, unless it is an integration test.
```

Comment:
Overall, this will need a little more work to be actually useful. A question I immediately have when reading this new doc is: Okay, but what IS used for integration tests then?

Reply:
done